godot-core/src/builtin/variant/impls.rs

```diff
@@ -186,10 +186,26 @@ mod impls {
                 Vector3, Vector3i, Vector3Axis, Vector4, Vector4i, Rect2, Rect2i, Plane, Quaternion, Aabb, Basis, Projection, Color, Rid
             }
         );
-
+        
+        impl<T: crate::meta::ArrayElement> crate::meta::sealed::Sealed for crate::task::ThreadConfined<crate::builtin::Array<T>> {}
+        
+        unsafe impl<T: crate::meta::ArrayElement> crate::task::DynamicSend for crate::task::ThreadConfined<crate::builtin::Array<T>> {
+            type Inner = crate::builtin::Array<T>;
+            fn extract_if_safe(self) -> Option<Self::Inner> {
+                self.extract()
+            }
+        }
+        
+        impl<T: crate::meta::ArrayElement> crate::task::IntoDynamicSend for crate::builtin::Array<T> {
+            type Target = crate::task::ThreadConfined<crate::builtin::Array<T>>;
+            fn into_dynamic_send(self) -> Self::Target {
+                crate::task::ThreadConfined::new(self)
+            }
+        }
```

Comment:
These symbols could probably all be imported for readability (usually IDEs allow this in one click, e.g. Alt+Enter in CLion/RustRover).

The macro doesn't import them so it stays independent of `use` statements surrounding it.